storage/innobase/xtrabackup/src/backup_copy.cc

```diff
@@ -1603,6 +1603,9 @@ backup_finish()
 		if (file_exists("ib_lru_dump")) {
 			copy_file(ds_data, "ib_lru_dump", "ib_lru_dump", 0);
 		}
+		if (file_exists("ddl_log.log")) {
+			copy_file(ds_data, "ddl_log.log", "ddl_log.log", 0);
+		}
```

Comment:
One doubt. So this file is parsed/used by the server and not by PXB full/incremental backups right?

Reply:
correct, server uses this file at `execute_ddl_log_recovery()` to decide if it has to rollback any multi table/patition DDL.

Reply:
Ok and testcase is hard to simulate?

Reply:
@satya-bodapati - there you go https://pxb.cd.percona.com/job/percona-xtrabackup-2.4-param-medium/104/

Reply:
@altmannmarcelo Good work on the testcase!